spec/facets/DataQualityMetricsInputDatasetFacet.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://openlineage.io/spec/facets/1-0-0/DataQualityMetricsInputDatasetFacet.json",
+  "$id": "https://openlineage.io/spec/facets/1-1-0/DataQualityMetricsInputDatasetFacet.json",
```

Comment:
I'd rather make it 1-0-1 bump according to https://github.com/OpenLineage/OpenLineage/blob/main/spec/Versioning.md#L29